data_generators/generators.py

```diff
@@ -82,7 +82,7 @@ class RandomMeeting(Printable):
     def __init__(self, mentee_id, mentor_id):
         self.meeting_id = generate_uuid(16)
         self.meeting_topic = choice(topics)
-        self.meeting_start_time = random_datetime(datetime(2022, 8, 1), datetime(2022, 8, 2))
+        self.meeting_start_time = random_datetime(datetime(2022, 1, 1), datetime(2022, 12, 30))
```

Comment:
Much better, and will make a visualization of meeting times much, **much** better. (as previously written it would have squished them all into one big spike.)

Reply:
Oh shoot. This is nitpicky. But December goes to 31 days...

Reply:
I can change it if that is necessary.